cmd/ncproxy/ncproxy.go

```diff
@@ -177,6 +259,43 @@ func (s *grpcService) CreateNetwork(ctx context.Context, req *ncproxygrpc.Create
 	}, nil
 }
 
+func constructEndpointPolicies(req *ncproxygrpc.CreateEndpointRequest) (*[]hcn.EndpointPolicy, error) {
```

Comment:
Any reason to return as a pointer?

Reply:
Not really. There may have been at one point, but I'll remove this.